### ISSUE TITLE
Debian GNU Linux 10.5 [go get] build failed

### ISSUE THREAD

**Issue by basilex** [kyleconroy/sqlc#717]
Hi!
Below is the problem during build process.

$ go get -u github.com/kyleconroy/sqlc/cmd/sqlc

github.com/kyleconroy/sqlc/internal/engine/dolphin
../kyleconroy/sqlc/internal/engine/dolphin/convert.go:343:14: n.Offset undefined (type *"github.com/pingcap/parser/ast".FuncCallExpr has no field or method Offset)

Opinions? Help please! Alexander

**Comment by johnatannvmd** [comment]
go version go1.15.2 linux/amd64

v1.4.0 ends with
```
# github.com/kyleconroy/sqlc/internal/mysql
../../go/pkg/mod/github.com/kyleconroy/sqlc@v1.4.0/internal/mysql/param.go:113:8: undefined: sqlparser.ParenExpr
```

go get github.com/kyleconroy/sqlc/cmd/sqlc@v1.3.0 - works as expected

**Comment by basilex** [comment]
> go version go1.15.2 linux/amd64
> 
> v1.4.0 ends with
> 
> ```
> # github.com/kyleconroy/sqlc/internal/mysql
> ../../go/pkg/mod/github.com/kyleconroy/sqlc@v1.4.0/internal/mysql/param.go:113:8: undefined: sqlparser.ParenExpr
> ```
> 
> go get github.com/kyleconroy/sqlc/cmd/sqlc@v1.3.0 - works as expected

ok, thanx for your help...)

**Comment by johnatannvmd** [comment]
Looks like `go get` builds 1.3.0 if it runs out of the project directory(with out go.mod file).
```
$ go get -v -u github.com/kyleconroy/sqlc/cmd/sqlc
github.com/kyleconroy/sqlc (download)
get "gopkg.in/yaml.v3": found meta tag get.metaImport{Prefix:"gopkg.in/yaml.v3", VCS:"git", RepoRoot:"https://gopkg.in/yaml.v3"} at //gopkg.in/yaml.v3?go-get=1
gopkg.in/yaml.v3 (download)
github.com/davecgh/go-spew (download)
github.com/pingcap/parser (download)
github.com/pingcap/errors (download)
github.com/pingcap/log (download)
get "go.uber.org/zap": found meta tag get.metaImport{Prefix:"go.uber.org/zap", VCS:"git", RepoRoot:"https://github.com/uber-go/zap"} at //go.uber.org/zap?go-get=1
go.uber.org/zap (download)
get "go.uber.org/atomic": found meta tag get.metaImport{Prefix:"go.uber.org/atomic", VCS:"git", RepoRoot:"https://github.com/uber-go/atomic"} at //go.uber.org/atomic?go-get=1
go.uber.org/atomic (download)
get "go.uber.org/multierr": found meta tag get.metaImport{Prefix:"go.uber.org/multierr", VCS:"git", RepoRoot:"https://github.com/uber-go/multierr"} at //go.uber.org/multierr?go-get=1
go.uber.org/multierr (download)
get "go.uber.org/zap/internal/bufferpool": found meta tag get.metaImport{Prefix:"go.uber.org/zap", VCS:"git", RepoRoot:"https://github.com/uber-go/zap"} at //go.uber.org/zap/internal/bufferpool?go-get=1
get "go.uber.org/zap/internal/bufferpool": verifying non-authoritative meta tag
get "go.uber.org/zap/buffer": found meta tag get.metaImport{Prefix:"go.uber.org/zap", VCS:"git", RepoRoot:"https://github.com/uber-go/zap"} at //go.uber.org/zap/buffer?go-get=1
get "go.uber.org/zap/buffer": verifying non-authoritative meta tag
get "go.uber.org/zap/zapcore": found meta tag get.metaImport{Prefix:"go.uber.org/zap", VCS:"git", RepoRoot:"https://github.com/uber-go/zap"} at //go.uber.org/zap/zapcore?go-get=1
get "go.uber.org/zap/zapcore": verifying non-authoritative meta tag
get "go.uber.org/zap/internal/color": found meta tag get.metaImport{Prefix:"go.uber.org/zap", VCS:"git", RepoRoot:"https://github.com/uber-go/zap"} at //go.uber.org/zap/internal/color?go-get=1
get "go.uber.org/zap/internal/color": verifying non-authoritative meta tag
get "go.uber.org/zap/internal/exit": found meta tag get.metaImport{Prefix:"go.uber.org/zap", VCS:"git", RepoRoot:"https://github.com/uber-go/zap"} at //go.uber.org/zap/internal/exit?go-get=1
get "go.uber.org/zap/internal/exit": verifying non-authoritative meta tag
get "gopkg.in/natefinch/lumberjack.v2": found meta tag get.metaImport{Prefix:"gopkg.in/natefinch/lumberjack.v2", VCS:"git", RepoRoot:"https://gopkg.in/natefinch/lumberjack.v2"} at //gopkg.in/natefinch/lumberjack.v2?go-get=1
gopkg.in/natefinch/lumberjack.v2 (download)
get "golang.org/x/text/encoding": found meta tag get.metaImport{Prefix:"golang.org/x/text", VCS:"git", RepoRoot:"https://go.googlesource.com/text"} at //golang.org/x/text/encoding?go-get=1
get "golang.org/x/text/encoding": verifying non-authoritative meta tag
golang.org/x/text (download)
get "golang.org/x/text/encoding/internal/identifier": found meta tag get.metaImport{Prefix:"golang.org/x/text", VCS:"git", RepoRoot:"https://go.googlesource.com/text"} at //golang.org/x/text/encoding/internal/identifier?go-get=1
get "golang.org/x/text/encoding/internal/identifier": verifying non-authoritative meta tag
get "golang.org/x/text/transform": found meta tag get.metaImport{Prefix:"golang.org/x/text", VCS:"git", RepoRoot:"https://go.googlesource.com/text"} at //golang.org/x/text/transform?go-get=1
get "golang.org/x/text/transform": verifying non-authoritative meta tag
get "golang.org/x/text/encoding/charmap": found meta tag get.metaImport{Prefix:"golang.org/x/text", VCS:"git", RepoRoot:"https://go.googlesource.com/text"} at //golang.org/x/text/encoding/charmap?go-get=1
get "golang.org/x/text/encoding/charmap": verifying non-authoritative meta tag
get "golang.org/x/text/encoding/internal": found meta tag get.metaImport{Prefix:"golang.org/x/text", VCS:"git", RepoRoot:"https://go.googlesource.com/text"} at //golang.org/x/text/encoding/internal?go-get=1
get "golang.org/x/text/encoding/internal": verifying non-authoritative meta tag
get "golang.org/x/text/encoding/japanese": found meta tag get.metaImport{Prefix:"golang.org/x/text", VCS:"git", RepoRoot:"https://go.googlesource.com/text"} at //golang.org/x/text/encoding/japanese?go-get=1
get "golang.org/x/text/encoding/japanese": verifying non-authoritative meta tag
get "golang.org/x/text/encoding/korean": found meta tag get.metaImport{Prefix:"golang.org/x/text", VCS:"git", RepoRoot:"https://go.googlesource.com/text"} at //golang.org/x/text/encoding/korean?go-get=1
get "golang.org/x/text/encoding/korean": verifying non-authoritative meta tag
get "golang.org/x/text/encoding/simplifiedchinese": found meta tag get.metaImport{Prefix:"golang.org/x/text", VCS:"git", RepoRoot:"https://go.googlesource.com/text"} at //golang.org/x/text/encoding/simplifiedchinese?go-get=1
get "golang.org/x/text/encoding/simplifiedchinese": verifying non-authoritative meta tag
get "golang.org/x/text/encoding/traditionalchinese": found meta tag get.metaImport{Prefix:"golang.org/x/text", VCS:"git", RepoRoot:"https://go.googlesource.com/text"} at //golang.org/x/text/encoding/traditionalchinese?go-get=1
get "golang.org/x/text/encoding/traditionalchinese": verifying non-authoritative meta tag
get "golang.org/x/text/encoding/unicode": found meta tag get.metaImport{Prefix:"golang.org/x/text", VCS:"git", RepoRoot:"https://go.googlesource.com/text"} at //golang.org/x/text/encoding/unicode?go-get=1
get "golang.org/x/text/encoding/unicode": verifying non-authoritative meta tag
get "golang.org/x/text/internal/utf8internal": found meta tag get.metaImport{Prefix:"golang.org/x/text", VCS:"git", RepoRoot:"https://go.googlesource.com/text"} at //golang.org/x/text/internal/utf8internal?go-get=1
get "golang.org/x/text/internal/utf8internal": verifying non-authoritative meta tag
get "golang.org/x/text/runes": found meta tag get.metaImport{Prefix:"golang.org/x/text", VCS:"git", RepoRoot:"https://go.googlesource.com/text"} at //golang.org/x/text/runes?go-get=1
get "golang.org/x/text/runes": verifying non-authoritative meta tag
github.com/lfittl/pg_query_go (download)
github.com/antlr/antlr4 (download)
# cd /home/johnatan/go/src/github.com/antlr/antlr4; git pull --ff-only
From https://github.com/antlr/antlr4
 * [new branch]          fix-javascript-perf-issue -> origin/fix-javascript-perf-issue
 * [new branch]          fix-template-issue -> origin/fix-template-issue
   621b933c7..e73f72be7  master             -> origin/master
You are not currently on a branch.
Please specify which branch you want to merge with.
See git-pull(1) for details.

    git pull <remote> <branch>

package github.com/antlr/antlr4/runtime/Go/antlr: exit status 1
github.com/jinzhu/inflection (download)
get "vitess.io/vitess/go/vt/sqlparser": found meta tag get.metaImport{Prefix:"vitess.io/vitess", VCS:"git", RepoRoot:"https://github.com/vitessio/vitess"} at //vitess.io/vitess/go/vt/sqlparser?go-get=1
get "vitess.io/vitess/go/vt/sqlparser": verifying non-authoritative meta tag
vitess.io/vitess (download)
get "vitess.io/vitess/go/bytes2": found meta tag get.metaImport{Prefix:"vitess.io/vitess", VCS:"git", RepoRoot:"https://github.com/vitessio/vitess"} at //vitess.io/vitess/go/bytes2?go-get=1
get "vitess.io/vitess/go/bytes2": verifying non-authoritative meta tag
get "vitess.io/vitess/go/sqltypes": found meta tag get.metaImport{Prefix:"vitess.io/vitess", VCS:"git", RepoRoot:"https://github.com/vitessio/vitess"} at //vitess.io/vitess/go/sqltypes?go-get=1
get "vitess.io/vitess/go/sqltypes": verifying non-authoritative meta tag
github.com/golang/protobuf (download)
get "google.golang.org/protobuf/encoding/prototext": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/encoding/prototext?go-get=1
get "google.golang.org/protobuf/encoding/prototext": verifying non-authoritative meta tag
google.golang.org/protobuf (download)
get "google.golang.org/protobuf/encoding/protowire": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/encoding/protowire?go-get=1
get "google.golang.org/protobuf/encoding/protowire": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/errors": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/errors?go-get=1
get "google.golang.org/protobuf/internal/errors": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/detrand": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/detrand?go-get=1
get "google.golang.org/protobuf/internal/detrand": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/encoding/messageset": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/encoding/messageset?go-get=1
get "google.golang.org/protobuf/internal/encoding/messageset": verifying non-authoritative meta tag
get "google.golang.org/protobuf/reflect/protoreflect": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/reflect/protoreflect?go-get=1
get "google.golang.org/protobuf/reflect/protoreflect": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/pragma": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/pragma?go-get=1
get "google.golang.org/protobuf/internal/pragma": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/encoding/text": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/encoding/text?go-get=1
get "google.golang.org/protobuf/internal/encoding/text": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/flags": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/flags?go-get=1
get "google.golang.org/protobuf/internal/flags": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/strs": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/strs?go-get=1
get "google.golang.org/protobuf/internal/strs": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/genid": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/genid?go-get=1
get "google.golang.org/protobuf/internal/genid": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/order": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/order?go-get=1
get "google.golang.org/protobuf/internal/order": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/set": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/set?go-get=1
get "google.golang.org/protobuf/internal/set": verifying non-authoritative meta tag
get "google.golang.org/protobuf/proto": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/proto?go-get=1
get "google.golang.org/protobuf/proto": verifying non-authoritative meta tag
get "google.golang.org/protobuf/reflect/protoregistry": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/reflect/protoregistry?go-get=1
get "google.golang.org/protobuf/reflect/protoregistry": verifying non-authoritative meta tag
get "google.golang.org/protobuf/runtime/protoiface": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/runtime/protoiface?go-get=1
get "google.golang.org/protobuf/runtime/protoiface": verifying non-authoritative meta tag
get "google.golang.org/protobuf/runtime/protoimpl": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/runtime/protoimpl?go-get=1
get "google.golang.org/protobuf/runtime/protoimpl": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/filedesc": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/filedesc?go-get=1
get "google.golang.org/protobuf/internal/filedesc": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/descfmt": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/descfmt?go-get=1
get "google.golang.org/protobuf/internal/descfmt": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/descopts": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/descopts?go-get=1
get "google.golang.org/protobuf/internal/descopts": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/encoding/defval": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/encoding/defval?go-get=1
get "google.golang.org/protobuf/internal/encoding/defval": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/filetype": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/filetype?go-get=1
get "google.golang.org/protobuf/internal/filetype": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/impl": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/impl?go-get=1
get "google.golang.org/protobuf/internal/impl": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/encoding/tag": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/encoding/tag?go-get=1
get "google.golang.org/protobuf/internal/encoding/tag": verifying non-authoritative meta tag
get "google.golang.org/protobuf/internal/version": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/internal/version?go-get=1
get "google.golang.org/protobuf/internal/version": verifying non-authoritative meta tag
get "vitess.io/vitess/go/hack": found meta tag get.metaImport{Prefix:"vitess.io/vitess", VCS:"git", RepoRoot:"https://github.com/vitessio/vitess"} at //vitess.io/vitess/go/hack?go-get=1
get "vitess.io/vitess/go/hack": verifying non-authoritative meta tag
get "vitess.io/vitess/go/vt/proto/query": found meta tag get.metaImport{Prefix:"vitess.io/vitess", VCS:"git", RepoRoot:"https://github.com/vitessio/vitess"} at //vitess.io/vitess/go/vt/proto/query?go-get=1
get "vitess.io/vitess/go/vt/proto/query": verifying non-authoritative meta tag
get "vitess.io/vitess/go/vt/proto/topodata": found meta tag get.metaImport{Prefix:"vitess.io/vitess", VCS:"git", RepoRoot:"https://github.com/vitessio/vitess"} at //vitess.io/vitess/go/vt/proto/topodata?go-get=1
get "vitess.io/vitess/go/vt/proto/topodata": verifying non-authoritative meta tag
get "vitess.io/vitess/go/vt/proto/vttime": found meta tag get.metaImport{Prefix:"vitess.io/vitess", VCS:"git", RepoRoot:"https://github.com/vitessio/vitess"} at //vitess.io/vitess/go/vt/proto/vttime?go-get=1
get "vitess.io/vitess/go/vt/proto/vttime": verifying non-authoritative meta tag
get "vitess.io/vitess/go/vt/proto/vtrpc": found meta tag get.metaImport{Prefix:"vitess.io/vitess", VCS:"git", RepoRoot:"https://github.com/vitessio/vitess"} at //vitess.io/vitess/go/vt/proto/vtrpc?go-get=1
get "vitess.io/vitess/go/vt/proto/vtrpc": verifying non-authoritative meta tag
get "vitess.io/vitess/go/vt/vterrors": found meta tag get.metaImport{Prefix:"vitess.io/vitess", VCS:"git", RepoRoot:"https://github.com/vitessio/vitess"} at //vitess.io/vitess/go/vt/vterrors?go-get=1
get "vitess.io/vitess/go/vt/vterrors": verifying non-authoritative meta tag
get "golang.org/x/net/context": found meta tag get.metaImport{Prefix:"golang.org/x/net", VCS:"git", RepoRoot:"https://go.googlesource.com/net"} at //golang.org/x/net/context?go-get=1
get "golang.org/x/net/context": verifying non-authoritative meta tag
golang.org/x/net (download)
get "google.golang.org/grpc/codes": found meta tag get.metaImport{Prefix:"google.golang.org/grpc", VCS:"git", RepoRoot:"https://github.com/grpc/grpc-go"} at //google.golang.org/grpc/codes?go-get=1
get "google.golang.org/grpc/codes": verifying non-authoritative meta tag
google.golang.org/grpc (download)
get "google.golang.org/grpc/status": found meta tag get.metaImport{Prefix:"google.golang.org/grpc", VCS:"git", RepoRoot:"https://github.com/grpc/grpc-go"} at //google.golang.org/grpc/status?go-get=1
get "google.golang.org/grpc/status": verifying non-authoritative meta tag
get "google.golang.org/genproto/googleapis/rpc/status": found meta tag get.metaImport{Prefix:"google.golang.org/genproto", VCS:"git", RepoRoot:"https://github.com/googleapis/go-genproto"} at //google.golang.org/genproto/googleapis/rpc/status?go-get=1
get "google.golang.org/genproto/googleapis/rpc/status": verifying non-authoritative meta tag
google.golang.org/genproto (download)
get "google.golang.org/protobuf/types/known/anypb": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/types/known/anypb?go-get=1
get "google.golang.org/protobuf/types/known/anypb": verifying non-authoritative meta tag
get "google.golang.org/grpc/internal/status": found meta tag get.metaImport{Prefix:"google.golang.org/grpc", VCS:"git", RepoRoot:"https://github.com/grpc/grpc-go"} at //google.golang.org/grpc/internal/status?go-get=1
get "google.golang.org/grpc/internal/status": verifying non-authoritative meta tag
get "google.golang.org/protobuf/types/known/durationpb": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/types/known/durationpb?go-get=1
get "google.golang.org/protobuf/types/known/durationpb": verifying non-authoritative meta tag
get "google.golang.org/protobuf/types/known/timestamppb": found meta tag get.metaImport{Prefix:"google.golang.org/protobuf", VCS:"git", RepoRoot:"https://go.googlesource.com/protobuf"} at //google.golang.org/protobuf/types/known/timestamppb?go-get=1
get "google.golang.org/protobuf/types/known/timestamppb": verifying non-authoritative meta tag
get "vitess.io/vitess/go/vt/log": found meta tag get.metaImport{Prefix:"vitess.io/vitess", VCS:"git", RepoRoot:"https://github.com/vitessio/vitess"} at //vitess.io/vitess/go/vt/log?go-get=1
get "vitess.io/vitess/go/vt/log": verifying non-authoritative meta tag
github.com/golang/glog (download)
get "vitess.io/vitess/go/vt/sysvars": found meta tag get.metaImport{Prefix:"vitess.io/vitess", VCS:"git", RepoRoot:"https://github.com/vitessio/vitess"} at //vitess.io/vitess/go/vt/sysvars?go-get=1
get "vitess.io/vitess/go/vt/sysvars": verifying non-authoritative meta tag
get "vitess.io/vitess/go/vt/vtgate/evalengine": found meta tag get.metaImport{Prefix:"vitess.io/vitess", VCS:"git", RepoRoot:"https://github.com/vitessio/vitess"} at //vitess.io/vitess/go/vt/vtgate/evalengine?go-get=1
get "vitess.io/vitess/go/vt/vtgate/evalengine": verifying non-authoritative meta tag
github.com/spf13/cobra (download)
github.com/spf13/pflag (download)

$ sqlc version
v1.3.0
```

**Comment by johnatannvmd** [comment]
moreover, clone the repo and simply run `go build cmd/sqlc/main.go` for v1.5.0 tag works well

**Comment by basilex** [comment]
> moreover, I had cloned the repo, checked out the `v1.5.0` tag, and then just ran `go build cmd/sqlc/main.go`. Binary built without a hitch

Thanx a lot! U helped me!!

**Comment by kyleconroy** [comment]
@basilex sorry for the build failure. I current am forced to use a fork of the MySQL parser, causing this build failure. The steps to fix it can be found in #654. I'm hoping that the necessary changes will be upstreamed soon, meaning that I can stop using my fork.

**Comment by basilex** [comment]
> @basilex sorry for the build failure. I currently am forced to use a fork of the MySQL parser, causing this build failure. The steps to fix it can be found in #654. I'm hoping that the necessary changes will be upstreamed soon, meaning that I can stop using my fork.

Thanx a lot! Glad to hear such things))
